app/demo-rv64-qemu-virt/app.toml

```diff
@@ -35,6 +35,7 @@ priority = 1
 stacksize = 512
 start = true
 features = ["semihosting"]
+exits = true
```

Comment:
Is this meant to be declared for all one-shot tasks?.  Maybe just changing the name/intention of it would help.  We could classify the tasks as "dont_kill", and the supervisor will run as long as one of the "dont_kill" tasks are still alive?

Reply:
Yes, I need to go back and make sure this is declared for all one-shot tasks.  I think we should try and avoid encoding intent for our particular use-case here though.

Previously (before the introduction of one-shot tasks) it would have been an error for a task to exit or return from main. (That should still be considered the case for non one-shot tasks as pointed out by @furquan-rivos in an earlier comment).

This 'exits' attribute simply indicates that this particular task's behavior is such that it may exit/return-from-main and that this is to be considered a deliberate act (not an error condition).

It's up to the supervisor of any given app (which may not necessarily be jefe) to decide what, if anything, to do with this additional information.  Some supervisors may not care, some may restart the task, and others may decide to poweroff (or affect some other state state change) once all 'exits' tasks have completed.

I'd prefer to simply keep the 'exits' attribute as a simple label of "this task exits" rather than trying to convey any sort of potential intent here.